docs/docs/getting-started.md

```diff
@@ -101,8 +101,47 @@ BigDL provides a convinient way to monitor/visualize your training progress. It
 ## **Tuning**
 
 There're several strategies that may be useful when tuning an optimization. 
+ * Change the learning Rate Schedule in SGD. Refer to [SGD docs](APIdocs/Optimizers/Optim-Methods.md#sgd) for details. 
+ * If overfit is seen, try use Regularization. Refer to [Regularizers](APIdocs/Regularizers.md). 
+ * Try change the initialization methods. Refer to [Initailizers](APIdocs/Initializers.md).
+ * Try Adam or Adagrad at the first place. If they can't achive a good score, use SGD and find a proper learning rate schedule - it usually takes time, though. RMSProp is recommended for RNN models. Refer to [Optimization Algorithms](APIdocs/Optimizers/Optim-Methods.md) for a list of supported optimization methods. 
+ 
+ 
+## Use log4j properity file to investigate issue using Debug log
```

Comment:
move this section to https://bigdl-project.github.io/master/#ProgrammingGuide/optimization/

Reply:
## Performance Tunning
For performance investigation, BigDL records the time-consuming distribution on each node for each step(e.g. sync weight, computing). The information can be displayed in the driver log. By default, it is suspended. To turn it on, please follow these steps:

1. Prepare a log4j property file
```
# Root logger option
log4j.rootLogger=INFO, stdout

# Direct log messages to stdout
log4j.appender.stdout=org.apache.log4j.ConsoleAppender
log4j.appender.stdout.Target=System.out
log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
log4j.appender.stdout.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n

log4j.logger.com.intel.analytics.bigdl.optim=DEBUG,myappender
log4j.addivity.com.intel.analytics.bigdl.optim=false

log4j.appender.myappender=org.apache.log4j.ConsoleAppender
log4j.appender.myappender.Target=System.out
log4j.appender.myappender.layout=org.apache.log4j.PatternLayout
log4j.appender.myappender.layout.ConversionPattern=%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %m%n
```

2. Add an option to your spark-submit command
```
--conf "spark.driver.extraJavaOptions=-Dlog4j.configuration=file:where_is_your_log4j_file."
```